vote_text_drafts/clarify_RPIP-4.md

```diff
@@ -0,0 +1,42 @@
+> Vote options:
+> - All nodes shall be eligible voters
+> - Nodes w/RPL ≥10% borrowed ETH shall be eligible voters
+> - Abstain
+
+# Clarify Voting Criteria (RPIP-4) [eligible voters: all nodes]
+
```

Comment:
Would recommend having sentence summary that will appear here:

![image](https://github.com/Valdorff/rp-thoughts/assets/53664591/b61d2f0c-0604-4079-a511-25232b2dd84e)

Just for general usability of snapshot space.